### DISCUSSION TITLE
Endpoint for version information

### DISCUSSION THREAD

**Discussion by larsl-net** [dani-garcia/vaultwarden#2432] [Ideas]
I am currently working on the setup of [Hymenaios](https://github.com/hymenaios-io/Hymenaios) as a solution for release monitoring. 

For vaultwarden I am looking for a way to get the current deployed version. 
So far I haven't found anything that supports Basic Auth or Api Keys (Header Based Auth) where I can get the version, over JSON or with RegEx. 

Is it possible to get a version endpoint in vaultwarden?

Which for example gives an output in this format?
```json
{
  "version": "1.24.0"
}
```

**Comment by jjlin** [comment]
Upstream provides the `/api/version` endpoint for the server version (e.g., https://vault.bitwarden.com/api/version).

#2433 implements this for Vaultwarden.
